packages/code-du-travail-frontend/src/api/__tests__/stats.test.ts

```diff
@@ -44,10 +44,7 @@ describe("Stats", () => {
       return Promise.reject("MATOMO IS DOWN");
     });
     const response = await request(server).get("/api/stats");
-    expect(response.status).toBe(404);
-    expect(response.body).toStrictEqual({
-      message: "No visit data and info data",
-    });
+    expect(response.status).toBe(500);
```

Comment:
Etrangement il y a ca qui a changé, on renvoit maintenant un status 500 au lieu de 404 mais ca me parait plus logique pour une erreur matomo is down

Reply:
c'est à cause de ça : https://github.com/SocialGouv/code-du-travail-numerique/pull/5971/files#r1647463707